pkg/tracing/opentelemetry/types.go

```diff
@@ -0,0 +1,47 @@
+package opentelemetry
+
+import (
+	"time"
+
+	"github.com/traefik/traefik/v2/pkg/types"
+)
+
+// Config provides configuration settings for an open-telemetry tracer.
+type Config struct {
+	GRPC *ConfigGRPC `description:"GRPC specific configuration for the OpenTelemetry collector." json:"grpc,omitempty" toml:"grpc,omitempty" yaml:"grpc,omitempty" export:"true" label:"allowEmpty" file:"allowEmpty"`
+
+	Compress bool              `description:"Enable compression on the sent data." json:"compress,omitempty" toml:"compress,omitempty" yaml:"compress,omitempty" export:"true"`
+	Endpoint string            `description:"Address of the collector endpoint." json:"endpoint,omitempty" toml:"endpoint,omitempty" yaml:"endpoint,omitempty"`
+	Headers  map[string]string `description:"Headers sent with payload." json:"headers,omitempty" toml:"headers,omitempty" yaml:"headers,omitempty" export:"true"`
+	Retry    *retry            `description:"The retry policy for transient errors that may occurs when exporting traces." json:"retry,omitempty" toml:"retry,omitempty" yaml:"retry,omitempty" export:"true"`
```

Comment:
Is there a specific intent that motivates to unexport the `retry` struct?
It's not gonna work for JSON unmarshalling.

```suggestion
	Retry    *Tetry            `description:"The retry policy for transient errors that may occurs when exporting traces." json:"retry,omitempty" toml:"retry,omitempty" yaml:"retry,omitempty" export:"true"`
```

Reply:
True, if there is JSON unmarshallling issues, this struct should be exported as `Retry` 😃